src/style.css

```diff
@@ -0,0 +1,133 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(/fonts/Avenir-Book.ttf);
+  font-style: normal;
+}
+@font-face {
+  font-family: "Avenir";
+  src: url(/fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+  font-style: normal;
+}
+
+body {
+  font-family: Avenir, serif;
+  margin: 20px 0;
+  box-sizing: border-box;
+}
+
+.container {
+  padding: 0 8px;
+  box-sizing: border-box;
+  width: 100%;
+}
+
+.search--bar--big {
+  position: relative;
+  width: 100%;
+  line-height: 68px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  box-sizing: border-box;
+}
+
+.search--bar--big:hover {
+  background: #ffff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.search--bar--big:focus {
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+}
+
+.input--big {
+  box-sizing: border-box;
+  width: 100%;
+  padding: 0;
+  padding-left: 62px;
+  outline: none;
+  line-height: 66px;
+  border: none;
+  border-radius: 4px;
+  font-weight: bold;
+  font-family: Avenir, serif;
+  color: #3d4e61;
+}
+
+.input--big::placeholder {
+  font-family: Avenir, serif;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+  font-weight: normal;
+}
+
+.input--big:focus {
+  background: linear-gradient(180deg, #ffff 0%, #f6f6f7 100%);
+}
+
+.icon {
+  position: absolute;
+  top: 24px;
+  left: 25px;
+  padding: 0;
+}
+
+/* .................Small search bar....................... */
+.search--bar--small {
+  position: relative;
+  margin-top: 20px;
+  width: 100%;
+  line-height: 38px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  box-sizing: border-box;
+}
+
+.search--bar--small:hover {
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.search--bar--small:focus {
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+}
+
+.input--small {
+  box-sizing: border-box;
+  width: 100%;
+  padding: 0;
+  padding-left: 33px;
+  outline: none;
+  line-height: 40px;
+  border: none;
+  border-radius: 4px;
+  font-weight: bold;
+  font-family: Avenir, serif;
+  color: #3d4e61;
+}
+
+.input--small:focus {
+  background: linear-gradient(180deg, #ffff 0%, #f6f6f7 100%);
+}
+
+.input--small::placeholder {
+  font-weight: 400;
+  font-family: Avenir, serif;
+  color: #3d4e61;
```

Comment:
you repeat yourself alot with these styles for small input, create one base class with similar styles (width, font, hover, focus, placeholder, border) and use it for both inputs